libs/live-config/src/providers/firebaseRemoteConfig/parser.ts

```diff
@@ -0,0 +1,40 @@
+import { Value } from "firebase/remote-config";
+import { ConfigInfo } from "../../schema";
+
+const stringParser = (value: unknown): string | undefined => {
+  return (value as Value).asString() || undefined;
+};
+
+const numberParser = (value: unknown): number | undefined => {
+  const numberValue = (value as Value).asNumber();
+  return numberValue !== 0 ? numberValue : undefined;
+};
+
+const booleanParser = (value: unknown): boolean | undefined => {
+  const valueExist = (value as Value).asString();
+  return valueExist ? (value as Value).asBoolean() : undefined;
+};
+
+const objectParser = (value: unknown): object | undefined => {
+  const stringValue = (value as Value).asString();
+  return stringValue ? JSON.parse(stringValue) : undefined;
+};
+
+export function parser(value: unknown, type: ConfigInfo["type"]) {
```

Comment:
I don't understand the need of this, the dev can just cast the result where config is needed